thunder/dynamo/splitter.py

```diff
@@ -175,7 +175,7 @@ def is_thunder_supported_partition(node: torch.fx.Node) -> bool:
             for n in graph_module.graph.nodes:
                 if n.op == "output":
                     for n in n.all_input_nodes:
-                        if "example_value" not in n.meta or n.meta["example_value"].grad_fn is None:
+                        if "example_value" not in n.meta or getattr(n.meta["example_value"], "grad_fn") is None:
```

Comment:
Maybe this is meant to be `getattr(n.meta["example_value"], "grad_fn", None)`?

Reply:
good catch